### ISSUE TITLE
Except for the first round of responses, all the others are garbled.   Could you provide the specific version number?

### ISSUE THREAD

**Issue by ShuoLiu0603** [oteroantoniogom/unsloth_vllm_triton_xformers_pytorch#1]
Unsloth 2025.6.2: Fast Qwen2 patching. Transformers: 4.52.4. vLLM: 0.8.3.dev11+g5d8e1c927.d20250618.cu128.
   \\   /|    NVIDIA GeForce RTX 5090. Num GPUs = 1. Max memory: 31.367 GB. Platform: Linux.
O^O/ \_/ \    Torch: 2.8.0.dev20250618+cu128. CUDA: 12.0. CUDA Toolkit: 12.8. Triton: 3.3.0
\        /    Bfloat16 = TRUE. FA [Xformers = 0.0.30+4fa0149.d20250618. FA2 = False]
 "-____-"     Free license: http://github.com/unslothai/unsloth
Unsloth: Fast downloading is enabled - ignore downloading bars which are red colored!
/root/shared-nvme/models/deepseekR1-Qwen-1.5B does not have a padding token! Will use pad_token = <|vision_pad|>.
Unsloth 2025.6.2 patched 28 layers with 28 QKV layers, 28 O layers and 28 MLP layers.
Unsloth: We now expect `per_device_train_batch_size` to be a multiple of `num_generations`.
We will change the batch size of 4 to the `num_generations` of 8
==((====))==  Unsloth - 2x faster free finetuning | Num GPUs used = 1
   \\   /|    Num examples = 7,473 | Num Epochs = 1 | Total steps = 700
O^O/ \_/ \    Batch size per device = 8 | Gradient accumulation steps = 2
\        /    Data Parallel GPUs = 1 | Total batch size (8 x 2 x 1) = 16
 "-____-"     Trainable parameters = 36,929,536/1,814,017,536 (2.04% trained)

![Image](https://github.com/user-attachments/assets/73cd5d11-4b0b-429f-866a-6ccd87d7aecd)
